syntax/pico8.vim

```diff
@@ -46,7 +46,7 @@ syn match  luaError "}"
 syn match  luaError "\<\%(end\|else\|elseif\|then\|until\|in\)\>"
 
 " Function declaration
-syn region luaFunctionBlock transparent matchgroup=luaFunction start="\<function\>" end="\<end\>" contains=ALLBUT,luaTodo,luaSpecial,luaCond,luaCondElseif,luaCondEnd,luaRepeat
```

Comment:
I don't really know what this is trying todo as `luaTODO` works inside a `function`.